lib/src/components/form/form.dart

```diff
@@ -852,7 +852,7 @@ class FormEntry<T> extends StatefulWidget {
   final Validator<T>? validator;
 
   const FormEntry(
-      {required FormKey<T> super.key, required this.child, this.validator});
+      {required FormKey super.key, required this.child, this.validator});
```

Comment:
Why is the type param removed?

Reply:
Not removed on purpose, i will add it back. Thanks for the note